pt/news/_posts/2017-12-14-ruby-2-5-0-rc1-released.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: news_post
+title: "Lançado Ruby 2.5.0-rc1"
+author: "naruse"
+translator: "jcserracampos"
+date: 2017-12-14 00:00:00 +0000
+lang: pt
+---
+
+Temos o prazer de anunciar o lançamento do Ruby 2.5.0-rc1.
+
+Ruby 2.5.0-rc1 é o primeiro candidato de lançamento do Ruby 2.5.0.
+Esta versão introduz algumas novas funcionalidades e melhorias de performance, por exemplo:
+
+## Novas Funcionalidades
+
+* Imprime _backtrace_ e mensagens de erros em ordem reversa se STDERR está inalterado e um tty.
+[Feature #8661] [experimental]
+
+* Pesquisa de constante de alto nível foi removida.  [Feature #11547]
+
+* rescue/else/ensure passam a ser permitidos em blocos do/end.  [Feature #12906]
+
+* Adiciona yield\_self.  [Feature #6721]
+
+## Melhorias de performance
+
+* Instrumentação dinâmica para hooks TracePoint ao invés de usar instrução
+  "trace" para evitar sobrecarga. [Feature #14104]
+
+* Performance de passagem de bloco usando parâmetros de blocos 
```

Comment:
There is another trailing space after `blocos`.

Reply:
I will be very thankful to you.
My code is beating me.

Reply:
Could you add me to `https://github.com/jcserracampos/www.ruby-lang.org` 😊 
Then I will push the change in this PR.

Reply:
Done!